### ISSUE TITLE
CSS3 Transition: Fade out sign in screen on state transition to overview

### ISSUE THREAD

**Issue by freemancw** [Infusion-Labs/portal#27]
When the user clicks on the `Let's Go` button in the sign in screen, three things should happen:

1. The sign in box should fade out over a 1 second interval.
2. The office lights should turn on over a 1 second interval.
3. After both (1) and (2) complete, initiate a ui-router `$state.go` call to transition to the `overview` state.

(1) and (2) should run in parallel. The office lights functionality is defined in `OfficeSceneService.turnOnLights(duration: number)`. This function should return an angular promise against which the client can program. Thus, the usage of the function will be:

```TypeScript
officeSceneService.turnOnLights(1000).then(function() {
    /* stuff */
});
``` 

##### Additional info and recommended reading

* https://docs.angularjs.org/guide/animations
* http://angular-ui.github.io/ui-router/site/#/api/ui.router.state.$state
* https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Using_CSS_transitions
* https://github.com/sole/tween.js/#usage
* https://docs.angularjs.org/api/ng/service/$q

**Comment by goodhanded** [comment]
90% the work on this was already done. Lol
